cmd/vic-machine/create/reservations.go

```diff
@@ -0,0 +1,169 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package create
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+
+	"github.com/urfave/cli"
+	"github.com/vmware/vic/pkg/errors"
+	"github.com/vmware/vic/pkg/trace"
+)
+
+func (c *Create) processReservations() error {
+	defer trace.End(trace.Begin(""))
+	if err := c.handleMemeoryReservation(c.memoryReservLimits); err != nil {
+		return err
+	}
+
+	if err := c.handleCPUReservation(c.cpuReservLimits); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (c *Create) handleCPUReservation(m string) error {
+	defer trace.End(trace.Begin(m))
+	if m == "" {
+		return nil
+	}
+
+	var errs []string
+
+	elements := strings.Split(m, ":")
```

Comment:
regex will be more efficient to find error, but error message will be only format error.
let me try that way to see the result.